### DISCUSSION TITLE
Account Flagged

### DISCUSSION THREAD

**Discussion by SickisM** [community/community#31829] [New to GitHub]
This is my first account and i just got insta flagged after doing a bug report to a game. It might have been because i used the browser thingy when clicking links redirects you to a default browser this is on an android phone btw. I was on reddit browsing how to pass a bug report to the game and it lead me here on github so i sign up to make a report then got flagged.

Sorry i made a new discussion but the other seemed abandoned since it got answered and the other people are seeking answers for this.

**Comment by queenofcorgis** [comment]
Hi @SickisM! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) The Community is unable to assist in matters like this so we'll be locking the thread!
